tests/expr17.py

```diff
@@ -0,0 +1,8 @@
+def f():
+    i: i32
+    i = 3
+    b: bool
+    b = bool(i)
+    print(b)
+
+f()
```

Comment:
Can we shift this to `integration_tests`? And add it in `integration_tests/CMakeLists.txt` with `c`, `cpp`, `cpython` and `llvm` backends?